build.sh

```diff
@@ -127,7 +128,7 @@ CMAKE_EXPORT_COMPILE_COMMANDS="-DCMAKE_EXPORT_COMPILE_COMMANDS=ON"
 LIBS_ONLY_BUILD=
 ALWAYS_YES=
 CCACHE_NAME=
-PYTHON2_BINARY=$(which python2.7 || which python2 || which python 2> /dev/null)
+PYTHON_BINARY=$(which python3 || which python || which python2.7 || which python2 || which python 2> /dev/null)
```

Comment:
`which python` is called twice - is that intentional? It seems if the first one fails, the second will also.